### ISSUE TITLE
Can't configure a client with one CPU per GPU.

### ISSUE THREAD

**Issue by bb30994** [FoldingAtHome/fah-issues#1566]
Your issue may already be reported!
Please search on the [issue tracker](./) before creating one.

## Your Environment
Recent beta FAHClient
* **F@H Software version**: 
* **Operating System**: 
* **Browser**: 

----
## Expected Behavior
I have a couple of systems with a 2-way (dual) CPU. I can't install two GPUs because the client forces me to leave room for a CPU slot.

----
## Current Behavior
Deleting the CPU slot causes a disruption when the client is restarted.

----
## Possible Solution (Optional)

----
## Steps To Reproduce

1.
2.
3.

----
## Context

----

**Comment by shorttack** [comment]
I don't think this is a bug. I have two Pentium (2 real cores) clients running Linux with two GPUs in each and no CPU clients. Works great and is quite economical.

However, getting the above to work is another story and not documents in one place. Therefore, it's problematic for new/inexperienced FAH users and especially those who are reticent to live-edit the config.xml file.

I believer the submitter is saying "I cannot add and run a GPU client within FAHcontrol."
1. Is the CPUs parameter set to -1 so the client can allocate whatever cores it wants from the pool of Available CPU threads - GPU CPUs? If the CPUs parameter is set to 2, no doubt there's no ability to add a GPU.
2. Try finishing a CPU work unit in FAHcontrol, then a) Pause the CPU client; b) add a GPU client and let it start. Check the log to see that it's running. then, c) delete the CPU client in config; e) add the second GPU slot in config.
3. Finish a CPU work unit in FAHcontrol, then a) copy and backup config.xml (File location depends on OS. See Forums). b) edit config.xml and replace "CPU" with "GPU". Save the config.xml file. Reboot.

What is needed for the growing community is a "How To" Guide beyond the existing foldingathome.org documentation.
